pubsub/mqtt/mqtt.go

```diff
@@ -216,7 +216,7 @@ func (m *mqttPubSub) Subscribe(req pubsub.SubscribeRequest, handler pubsub.Handl
 		token := m.consumer.SubscribeMultiple(
 			m.topics,
 			func(client mqtt.Client, mqttMsg mqtt.Message) {
-				mqttMsg.AutoAckOff()
+				// mqttMsg.AutoAckOff()
```

Comment:
Is this intentional?

Reply:
Hello, this should not be intentional, the source code should be restored